src/jit/codegenlegacy.cpp

```diff
@@ -5646,12 +5646,10 @@ void CodeGen::genCodeForQmark(GenTreePtr tree, regMaskTP destReg, regMaskTP best
             // If any candidates are not alive at the GT_QMARK node, then they
             // need to be spilled
 
-            VARSET_TP VARSET_INIT(compiler, rsLiveNow, compiler->compCurLife);
-            VARSET_TP VARSET_INIT_NOCOPY(rsLiveAfter, compiler->fgUpdateLiveSet(compiler->compCurLife,
-                                                                                compiler->compCurLifeTree, tree));
+            VARSET_VALARG_TP rsLiveNow(compiler->compCurLife);
```

Comment:
Looks like it is a naming problem.
As I understand that:
1. VARSET_VALARG_TP was created to replace "const VARSET_TP &" that was used in a function signature to pass an argument that could be read, but can't be changed. We do not need this type anywhere except arg lists.
2. If an argument can be changed, then we use "VARSET_TP &", fgComputeLifeCall is an example.

I don't see a problem in such declaration, when we create an argument not in argList, because it has a long name for example.
Compare:
```
VARSET_VALARG_TP gcrefVarsArg(getEmitter()->emitThisGCrefVars); 
getEmitter()->emitCreatePlaceholderIG(IGPT_EPILOG, block, gcrefVarsArg, gcrefRegsArg, byrefRegsArg, last); 
```
and 
`getEmitter()->emitCreatePlaceholderIG(IGPT_EPILOG, block, getEmitter()->emitThisGCrefVars, gcrefRegsArg, byrefRegsArg, last); `
I think the first variant is better.

I do not want to use VARSET_TP because there is a possibility that somebody in the future make a change to modify this variable without understanding that `compiler->compCurLife` will change too.